config/tests/samples/create/samples.go

```diff
@@ -177,13 +180,12 @@ func waitForReadySingleResource(t *Harness, u *unstructured.Unstructured) {
 		return
 	}
 
-	name := k8s.GetNamespacedName(u)
 	err := wait.PollImmediate(1*time.Second, 35*time.Minute, func() (done bool, err error) {
 		done = true
-		logger.V(2).Info("Testing to see if resource is ready", "kind", u.GetKind(), "name", u.GetName())
```

Comment:
(If you mean "why did we stop adding the values, it's because we add them to the logger above:

```
logger := log.FromContext(t.Ctx).WithValues("kind", gvk.Kind, "name", id.Name)
```
)